src/ChoiceBox.hpp

```diff
@@ -30,7 +30,7 @@ struct Choice
   std::string prereq = "";
 };
 
-class ChoiceBox : Box
+class ChoiceBox : public Box
```

Comment:
I forgot why adding public helps?

Reply:
I will refer you to [another SO post](https://stackoverflow.com/questions/860339/difference-between-private-public-and-protected-inheritance). It is necessary so that other code can call the base class methods (like the needsDraw() function).